docs/sources/mimir/release-notes/v2.9.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Grafana Mimir version 2.9 release notes"
+menuTitle: "V2.9 release notes"
+description: "Release notes for Grafana Mimir version 2.9"
+weight: 300
+---
+
+# Grafana Mimir version 2.9 release notes
+
+Grafana Labs is excited to announce version 2.9 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Experimental support for optimized series selection strategy** Mimir can now use an optimized strategy for fetching posting lists that can significantly reduce memory allocations in the store-gateway.
+- **Reduced CPU utilisation for some shuffle sharding scenarios** Mimir queriers will now use significantly less CPU in cases where shuffle sharding is enabled with a high number of shards.
+- **Reduced object storage API calls in compactors and rulers** Mimir 2.9 comes with optimizations that will reduce the amount of times compactors and rulers need to access rules stored in object storage.
+- **Experimental support for streaming chunks from ingester to querier** This is expected to greatly reduce querier memory consumption when evaluating queries that select a large number of chunks, because chunks streamed from the querier can now be read into memory as needed.
+
+### Helm chart improvements
+
+The Grafana Mimir and Grafana Enterprise Metrics Helm chart is now released independently. See the [Grafana Mimir Helm chart documentation](/docs/helm-charts/mimir-distributed/latest/).
+
+## Important changes
+
+In Grafana Mimir 2.9 we have removed the following previously deprecated or experimental metrics:
+
+- `cortex_bucket_store_chunk_pool_requested_bytes_total`
+- `cortex_bucket_store_chunk_pool_returned_bytes_total`
+
+The following configuration options are deprecated and will be removed in Grafana Mimir 2.11:
```

Comment:
There will be two more to add to this list (`-querier.iterators` and `-query.batch-iterators`) after https://github.com/grafana/mimir/pull/5114 is merged